larq_compute_engine/mlir/python/end2end_test.py

```diff
@@ -0,0 +1,76 @@
+import sys
+import pytest
+import larq as lq
+import subprocess
+import tensorflow as tf
+import numpy as np
+
+from larq_compute_engine.mlir.python.converter import convert_keras_model
+
+
+@pytest.fixture
+def model():
```

Comment:
I agree that we can test some larq zoo models.
However we should *also* test this simple model because it explicitly tests for some important things like relu fusion, zero-padding (test if relu's do *not* get fused here), one-padding etc.